packages/react-devtools-shared/src/devtools/views/Components/types.js

```diff
@@ -64,6 +64,14 @@ export type InspectedElementResponseType =
   | 'no-change'
   | 'not-found';
 
+// For now, let's only support a hard-coded set of plugins.
+type PluginType = 'stylex';
```

Comment:
should we share this type somewhere instead of defining it twice?

Reply:
Sure. I'll take a pass at tidying/drying this up.